app/src/main/res/values/strings.xml

```diff
@@ -67,6 +67,7 @@
     <string name="user_search">User Search</string>
     <string name="logout">Logout</string>
     <string name="editLocation">EDIT LOCATION</string>
+    <string name="no_progress">00%</string>
```

Comment:
Is there a reason you chose `00` instead of just `0`?

Reply:
Initially I wanted the placeholder to reflect the possibility of having a two digit number - I changed the textview to wrap contents instead though and automatically center so it no longer matters